docs/YML-CBuild-Format.md

```diff
@@ -33,13 +33,47 @@ File                | Description
 :-------------------|:----------------------
 `*.cbuild-idx.yml`  | Index file of all `*.cbuild.yml` build descriptions; contains also overall information for the application.
 `*.cbuild.yml`      | Build description of a single [`*.cproject.yml`](YML-Input-Format.md#project-file-structure) input file.
+`*.cbuild-pack.yml` | Pack descriptions recorded for the entire [`*.csolution.yml`](YML-Input-Format.md#project-file-structure) input file.
 
 The `*.cbuild.yml` output file has the following usage:
 
 - It contains all information for the build step of a project that is part of an application.
-- As it contains information about all software packs used including version information, this file can be used as `lock-file` to ensure that subsequent runs of `csolution` use the very same software packs.
 - It can be used as input file for a generator as it contains explicit information about source files and avoids the complexity of a pack data management at the generator level.
 
+The `*.cbuild-pack.yml` file has the following usage:
+
+- It contains strict list of software packs used including version information and ranges from `.csolution.yml` and all subsequent potentially included `.cproject.yml` and `.clayer.yml` files.
+- This file is used as `lock-file` to ensure that subsequent runs of `csolution` use the very same packs and pack versions on all computers / setups / runs / environments.
+
+### Pack locking
```

Comment:
Is it possible to document when the lock file is updated in the user workflow?

Reply:
@slhultgren, I have not looked at the source code, but my expectation would be that the update of the `<solution>.cbuild-pack.yml` would also be included in the `update-rte` command but not in any of the `list` commands.

Reply:
Clarified this in the updated docs